### PR TITLE
Laravel 7 support

### PR DESCRIPTION
**Pull request by wilburpowery**
@michaeldyrynda 🤠

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,11 +8,11 @@ matrix:
   fast_finish: true
   include:
     - php: 7.2
-      env: ILLUMINATE_VERSION=6.0.*
```
Comment (michaeldyrynda) [inline .travis.yml:11]: I think we should make these `7.*` now, or maybe even just `^7.0`

Comment (wilburpowery) [reply .travis.yml:11]: Totally agree. Fixed. Made them `^7.0`

**Comment by wilburpowery** [comment]
Rebased and squashed the commits to keep the history cleaner.

**Comment by michaeldyrynda** [comment]
Thanks!

**Pull request merged**